src/main/steps/application-within-proceedings/document-upload/postController.ts

```diff
@@ -118,6 +124,16 @@ export default class UploadDocumentController extends PostController<AnyObject>
     return true;
   }
 
+  private isExceedingMaxApplicationForms(isSupportingDocuments: any, req: any) {
```

Comment:
This method can be put in a common file and reused across document upload. Preferably in validation.ts

Reply:
moved the validation logic to validation.ts